streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java

```diff
@@ -507,7 +507,9 @@ public synchronized StreamsBuilder addGlobalStore(final StoreBuilder storeBuilde
      * of the input topic.
      * <p>
      * The provided {@link ProcessorSupplier} will be used to create an {@link ProcessorNode} that will receive all
-     * records forwarded from the {@link SourceNode}.
+     * records forwarded from the {@link SourceNode}. NOTE: you should not use this to insert transformed records into
```

Comment:
nit: remove `use this to`

Or `use the {@code Processor} to`

Reply:
Ack